contribute/style-guides/code-comments.md

```diff
@@ -1,45 +1,35 @@
-# Guidelines for code comments in grafana-\* packages
+# Guidelines for code comments in Grafana packages
 
-This document aims to give you some recommendation on how to add code comments to the exported code in the grafana packages.
-
-## Table of Contents
-
-1. [Add package description](#add-package-description)
-1. [Set stability of an API](#set-stability-of-an-api)
-1. [Deprecate an API](#deprecate-an-api)
-1. [Specify parameters](#specify-parameters)
-1. [Set return values](#set-return-values)
-
----
+This guide provides recommendations for adding code comments in `grafana-\*` packages.
 
 ## Add package description
 
 Each package has an overview explaining the overall responsibility and usage of the package.
 
-You can document this description with [`@packageDocumentation`](https://api-extractor.com/pages/tsdoc/tag_packagedocumentation/) tag.
+Use the [`@packageDocumentation`](https://api-extractor.com/pages/tsdoc/tag_packagedocumentation/) tag to document the package.
 
-Add this tag to the `<packageRoot>/src/index.ts` entry file to have one place for the package description.
+Add the `@packageDocumentation` tag to the `<packageRoot>/src/index.ts` entry file to have one place for the package description.
 
 ## Set stability of an API
 
-All `exported` apis from the package should have a release tag to indicate its stability.
+Add a release tag to all exported APIs from the package to indicate its stability.
 
-- [`@alpha`](https://api-extractor.com/pages/tsdoc/tag_alpha/) - early draft of api and will probably change.
-- [`@beta`](https://api-extractor.com/pages/tsdoc/tag_beta/) - close to being stable but might change.
-- [`@public`](https://api-extractor.com/pages/tsdoc/tag_public/) - ready for usage in production.
-- [`@internal`](https://api-extractor.com/pages/tsdoc/tag_internal/) - for internal use only.
+- [`@alpha`](https://api-extractor.com/pages/tsdoc/tag_alpha/) - denotes an early draft of API that will probably change.
+- [`@beta`](https://api-extractor.com/pages/tsdoc/tag_beta/) - denotes that the API is close to being stable but might change.
+- [`@public`](https://api-extractor.com/pages/tsdoc/tag_public/) - denotes that the API is ready for usage in production.
+- [`@internal`](https://api-extractor.com/pages/tsdoc/tag_internal/) - denotes that the API is for internal use only.
 
-### Main stability of APIs
+### Indicate main stability of APIs
 
-Add a tag to mark the stability of the whole exported `class/interface/function/type` etc.
+Add a tag to mark the stability of the whole exported `class/interface/function/type`, and other interfaces.
 
-Please place the `release tag` at the bottom of the comment to make it consistent among files and easier to read.
+Place the release tag at the bottom of the comment to make it consistent among files and easier to read.
```

Comment:
Should these be numbered steps?

Reply:
no, it wouldn't make any difference which order these steps are done in; however, on giving this section a second look i think these steps would be better as bullets